core/string/casecmp_spec.rb

```diff
@@ -117,4 +117,65 @@
       "B".casecmp(a).should == 1
     end
   end
+
+  ruby_version_is "2.4" do
```

Comment:
This block should be dedented once, so that `String#casecmp?` is no nested in `String#casecmp`.
You can check the result with `mspec -fs ...`.